public/testem/testem_client.js

```diff
@@ -266,6 +267,11 @@ var Testem = {
         case 'stop-run':
           self.emit('after-tests-complete');
           break;
+        default:
+          if (type.startsWith('testem:')) {
```

Comment:
`startsWith` is ES6 only, so to keep our browser code compat with older versions lets create a `startsWith` helper instead.

I believe the following should do the trick:

```js
function starsWith(str, search, pos) {
  return str.substr(!pos || pos < 0 ? 0 : +pos, search.length) === search;
}
```

Reply:
I made changes to  use `indexOf` for instance`'evnetName.indexOf('testem:') == 0` .